src/api/routes/index.js

```diff
@@ -1,10 +1,12 @@
 import express from 'express';
 import authRouter from './authRouter';
 import resetRouter from './password.reset';
+import articleRouter from './articles';
 
 const router = express();
 
 router.use('/users', authRouter);
 router.use('/password-reset', resetRouter);
+router.use(articleRouter);
```

Comment:
I think it makes more sense if you use `router.use('/articles', articleRouter);` @murediane